x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -69,26 +73,38 @@ interface MLInferenceProcessorsActions {
   setInferencePipelineConfiguration: (configuration: InferencePipelineConfiguration) => {
     configuration: InferencePipelineConfiguration;
   };
+  setPipelineSimulateBody: (simulateBody: string) => {
+    simulateBody: string;
+  };
+  setSimulatePipelineErrors(errors: string[]): { errors: string[] };
+  simulatePipeline: () => void;
+  simulatePipelineApiError(error: HttpError): HttpError;
```

Comment:
Please import/construct this from the API logic rather than typing it yourself. That avoids preventable mistakes.